typescript/sdk/src/hook/read.ts

```diff
@@ -1,5 +1,6 @@
 import { assert } from 'console';
 import { ethers, providers } from 'ethers';
+import { stringify as yamlStringify } from 'yaml';
```

Comment:
Yaml is currently a dev dep of the SDK, not a runtime dep, meaning this would fail if someone didn't happen to have the yaml lib bundled already. You can make it a runtime dep, it's small so nbd.

Reply:
good catch